src/kirby/components/loading-overlay/loading-overlay.component.tns.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, OnInit, Input, ViewChild, ElementRef, AfterViewInit } from '@angular/core';
+import { isAndroid } from 'tns-core-modules/ui/page/page';
+import { LayoutBase, View } from 'tns-core-modules/ui/layouts/layout-base';
+import { BehaviorSubject } from 'rxjs';
+
+@Component({
+  selector: 'kirby-loading-overlay',
+  templateUrl: './loading-overlay.component.html',
+  styleUrls: ['./loading-overlay.component.scss'],
+})
+export class LoadingOverlayComponent implements OnInit, AfterViewInit {
+  private isLoadingSubject = new BehaviorSubject(false);
+
+  private _isLoading: boolean;
+  public get isLoading(): boolean {
+    return this._isLoading;
+  }
+  @Input()
+  public set isLoading(v: boolean) {
+    this._isLoading = v;
+    this.isLoadingSubject.next(v);
+  }
+
+  @Input() public showBackdrop = true;
+
+  @ViewChild('overlayWrapperRef')
+  wrapperGrid: ElementRef;
+
+  constructor() {}
+
+  ngOnInit() {}
+
+  ngAfterViewInit(): void {
+    this.isLoadingSubject.subscribe((isLoading) => {
```

Comment:
We should not unsubscribe this subscription at some point?

Reply:
Unsubscribed on ondestroy now